applications/values.yaml

```diff
@@ -70,10 +70,15 @@ applications:
     jsonPointers:
     - /spec/template/spec/containers/0/image
 
-- name: omp-launcher
+- name: resource-dispatcher
   url: https://github.com/redhat-cop/tool-integrations.git
-  path: "argo-cd-bridges/gitlab/deploy"
-  ref: *launcherVersion
+  path: "resource-dispatcher/helm"
+  ref: *resourceDispatcherVersion
+  helmValues:
+    configMapName: resource-dispatcher-config
+    giveClusterAdmin: true
```

Comment:
why would this deployment need cluster-admin?

Reply:
Some sort of elevated privileges are needed for managing `argoproj.io/application` resources using the kube api. The bridge sort of _already_ had elevated privileges for managing app resources because it bound to Argo's service account, which I believe has cluster-admin... Thought that just setting it this way might be cleaner. Alternatively, I can do what that did and just bind to that same service account or we can set up a more specific one.